### ISSUE TITLE
Instagram Post issue

### ISSUE THREAD

**Issue by DareDevil2000** [mozilla/multi-account-containers#2640]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

Create a container go to--> Instagram --> click create---> select an image---> make it original -->Click next --> again click next--> now the problem happened [ The image become upside down ]
![screenshot_300](https://github.com/mozilla/multi-account-containers/assets/64276640/3f19d233-fa69-4d70-9094-a42882498d25)

### Actual behavior

The image becomes upside down .

### Expected behavior

The image should remain as it is.

### Additional informations

When i use the size outside of the container everything as expected but when i use it in a container the problem appears.
Also tried deleting and creating a new container, Still same issue.

**Comment by bakulf** [comment]
@DareDevil2000 can you reproduce the issue using a private browsing section?

**Comment by DareDevil2000** [comment]
Are you talking about "Private Window" ?

**Comment by bakulf** [comment]
Yes please.

**Comment by DareDevil2000** [comment]
@bakulf But can we use multi-account-containers in "Private Window" ?

**Comment by bakulf** [comment]
What I meant is to reproduce the issue in private browsing mode without containers. The underline technology is shared between containers and private browsing.

**Comment by DareDevil2000** [comment]
@bakulf Now i get it.

**Comment by DareDevil2000** [comment]
When using 'Private Window' mode, I didn't encounter any problems, so I suspected that an extension might be causing the issue. I disabled all the extensions, and the site worked fine without them. After turning them back on one by one, I finally identified the problematic extension: [CanvasBlocker](https://addons.mozilla.org/en-US/firefox/addon/canvasblocker/). This was the one causing the issue.
